src/components/ListOfPokemonControlButton.js

```diff
@@ -0,0 +1,22 @@
+import '../style/ListOfPokemonControlButton.css';
+
+export function ListOfPokemonControlButton({showPokemonListHTML, setShowPokemonListHTML}){
+
+  function changeHTMLshowPokemonListBtn(){
+    if(showPokemonListHTML === 'Show list of Pokemon' ){
```

Comment:
That's weird. Don't mix copy (Copy to po prostu szeroko pojęta treść.) with logic of the app.
If you want to have toggle button to show or hide list use boolean values.